### ISSUE TITLE
this.parse() is undefined

### ISSUE THREAD

**Issue by dim882** [wrapp-archive/validate.js#26]
I'm attempting to use moment.js for date validations, and I have a validation rule that's pretty much the same as the XDate example in the documentation:

```
    birthdate: {
      date: {
        latest: moment().subtract(18, 'years'),
        message: "^You need to be at least 18 years old"
      }
    },
```

but I get an "Uncaught TypeError: undefined is not a function validate.js:622". It appears it's attempting to call `#parse` where `this` is the `validators` object, not the `datetime` object, so it fails. I wonder if it's possible I'm calling `validate()` wrong and screwing up the context, but I can't find anything in my code that would cause that.

Anyway, I'll submit a pull request soon with a fix that works for me, but let me know if this is incorrect.

**Comment by ansman** [comment]
Hmm, `this` should refer to the `datetime` validator when called since validate.js calls each validator like this:

``` javascript
results.push({
  attribute: attr,
  error: validator.call(validator, value, validatorOptions, attr,
                        attributes)
});
```

Could you add a `console.log(this)` before line 622 and see what `this` refers to?

**Comment by ansman** [comment]
Ah, I found the issue and released [0.3.1](http://validatejs.org/)

**Comment by dim882** [comment]
Ah, I see. Thanks for getting the fix out so quickly!
